app_dart/test/request_handlers/github_webhook_test.dart

```diff
@@ -904,35 +904,6 @@ void main() {
         request.headers.set('X-GitHub-Event', 'pull_request');
       });
 
-      test('Exception is raised when no builders available', () async {
```

Comment:
This is no longer true as there's always at least one check we generate on presubmit